qualtran/bloqs/factoring/rsa/rsa_mod_exp.py

```diff
@@ -59,17 +61,17 @@ class ModExp(Bloq):
 
     Registers:
         exponent: The exponent
-        x [right]: The output register containing the result of the exponentiation
+        x: The output register containing the result of the exponentiation
```

Comment:
you didn't like my "[right]"? I think we should document sided registers somehow

Reply:
Oops, my bad. I forgot to add this back when I reconfigured mod exp to not be used in the phase estimation.